app/scripts/proactive/config.js

```diff
@@ -62,6 +62,50 @@ define(function () {
             'Script/flow' : [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
             'Script/clean' : [" ", "bash", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R"]
         },
+        extensions_to_languages: {
+            "sc": "scalaw",
+            "scala": "scalaw",
+            "groovy": "groovy",
+            "js": "javascript",
+            "py" : "python",
+            "rb": "ruby",
+            "ps": "powershell",
+            "r": "R",
+            "sh": "bash",
+            "yaml": "kubernetes",
+            "yml": "docker-compose",
+            "pl": "perl"
+        },
+        languages_to_extensions: {
+            "scalaw" : "sc",
```

Comment:
the canonical scala extension seems to be .scala. Could you double-check with gleb ?